package.json

```diff
@@ -19,7 +19,7 @@
     "fs-extra": "^4.0.2",
     "glob": "^7.1.2",
     "is-glob": "^4.0.0",
-    "loader-utils": "^0.2.15",
+    "loader-utils": "^1.1.0",
```

Comment:
This is strictly a 🏷  Major 😞 . We cpuld do it, if we find consensus about being a bit looser with semver here, since node =< 4 is EOL anyways 😛

Reply:
@michael-ciniawsky i think it is more fix then breaking changes because some interpolation don't works as expected

Reply:
And yes all that < node 4 is EOL :smile:

Reply:
I'm basically ok with it, but let's wait a bit what others say

Reply:
How about we skip the grey area on the breaking nature of loader-utils & just roll it in with defaults which is most definitely a major. We can publish to `@beta / @next / @purple_fluffie_bunnies` off the `#next` branch until everyone is satisfied with it. Also gives me a chance to test against angular/cli without blocking other PRs in master